### PR TITLE
allow lambda function to have "." in its filename

### PR DESCRIPTION
**Pull request by Newbie012**
Fixes an issue where a lambda filename is "x.lambda.ts"

Currently, it fails to locate the lambda file since it looks for `x.ts` instead of `x.lambda.ts`

### CODE REVIEW

**Comment by fwang** [comment]
Thanks @Newbie012, we are making some structural changes to the repo, will merge this in after those changes are merged.

**Review by 53jk1 — APPROVED** [review]

**Comment by thdxr** [comment]
We are transition this repo to https://github.com/sst/v2 and are closing this issue for now. If it is still relevant please reopen it there - sorry for the trouble

**Pull request closed without merging**
